components/molecules/ContributorListTableRow/contributor-list-table-row.tsx

```diff
@@ -89,14 +89,18 @@ const ContributorListTableRow = ({
             />
           )}
           <div className="w-[68%]">
-            <DevProfile
-              company={user?.company || getLastContributedRepo(data)}
-              username={login}
-              hasBorder={!contributor.author_login}
-            />
+            <DevProfile username={login} hasBorder={!contributor.author_login} />
           </div>
           <div className="w-[34%] text-normal text-light-slate-11  h-full">
-            <div className="flex gap-x-3">{<p>{getLastContributionDate(mergedPrs)}</p>}</div>
+            <div className="flex flex-col gap-x-3">
+              <p>{getLastContributionDate(mergedPrs)}</p>{" "}
+              <p
+                className="text-sm font-normal truncate whitespace-nowrap overflow-hidden overflow-ellipsis text-light-slate-9 md:hidden"
```

Comment:
This works, but the columns are getting too large in some cases due to a long org/project name and the ellipsis isn't kicking in, e.g.

![CleanShot 2023-11-13 at 15 15 03](https://github.com/open-sauced/app/assets/833231/a3047c20-7f17-4574-b513-eb4af031b5d5)

A max width would help with this.

I think longer term we need to revisit this grid, but for now, that should help with the misalignment.